ui/src/components/Feed/FeedPost.svelte

```diff
@@ -164,7 +164,6 @@
   };
   const showLess = () => {
     postContainer.classList.add('max-h-96');
-    postContainer.scrollIntoView();
```

Comment:
this was causing strange things to happen to the div. 
<img width="664" alt="Screenshot 2023-09-01 at 1 11 18 AM" src="https://github.com/worpet-bildet/portal/assets/24712874/9969cf6b-7ce1-42f1-868a-391a38a0ce71">

Reply:
it's due to my new assignment of this:postContainer, but i couldn't figure out a way to work around it. any ideas?

Reply:
It's hard to read an html diff this long on github, but my guess is that you're changing the height of the wrong div or have gotten the div hierarchy incorrect.

Reply:
couldn't figure this out. leaving scrolling out since this is a net positive. :/